recipes/yapsy/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "yapsy" %}
```

Comment:
You need to change this to `Yapsy` (note capitalization):

https://pypi.org/project/Yapsy/#files